lib/checkPredicates.ml

```diff
@@ -286,11 +295,14 @@ let ask_solver g lcs =
   in
   let res =
     match solver_res with
-    | `True -> Result.no (Pp.( !^ ) "Solver returned No.")
-    | `Unknown -> Result.unknown (Pp.( !^ ) "Solver returned Unknown.")
+    | `True -> Result.no (Pp.( !^ ) "The solver proved this resource is inconsistent.")
+    | `Unknown ->
+      Result.unknown (Pp.( !^ ) "The solver timed out trying to validate this resource.")
```

Comment:
You can either use `Pp.string` or do a local open around the whole expression `Pp.(!^”…”)`